File: app/src/main/java/com/example/qpony/Network/Model/Rates.java
package com.example.qpony.Network.Model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Rates {

    @SerializedName("USD")
    @Expose
    private String USD;

    @SerializedName("AUD")
    @Expose
    private String AUD;

    @SerializedName("CAD")
    @Expose
    private String CAD;

    @SerializedName("PLN")
    @Expose
    private String PLN;

    public String getUSD() {
        return USD;
    }

    public String getAUD() {
        return AUD;
    }

    public String getCAD() {
        return CAD;
    }

    public String getPLN() {
        return PLN;
    }

    public String getMXN() {
        return MXN;
    }

    @SerializedName("MXN")
    @Expose
    private String MXN;

}

File: app/src/main/java/com/example/qpony/Model/Rates.kt
package com.example.qpony.Model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Rates {

    @SerializedName("USD")
    @Expose
    val usd: Double? = null

    @SerializedName("AUD")
    @Expose
    val aud: Double? = null

    @SerializedName("CAD")
    @Expose
    val cad: Double? = null

    @SerializedName("PLN")
    @Expose
    val pln: Double? = null

    @SerializedName("MXN")
    @Expose
    val mxn: Double? = null

}
